test/moves/ohko.test.ts

```diff
@@ -0,0 +1,108 @@
+import { Abilities } from "#enums/abilities";
+import { MoveId } from "#enums/move-id";
+import { Species } from "#enums/species";
+import { GameManager } from "#test/testUtils/gameManager";
+import Phaser from "phaser";
+import { afterEach, beforeAll, beforeEach, describe, it, expect, vi } from "vitest";
+import { allMoves } from "#app/data/data-lists";
+
+describe("Moves - One Hit KO Moves", () => {
+  let phaserGame: Phaser.Game;
+  let game: GameManager;
+
+  beforeAll(() => {
+    phaserGame = new Phaser.Game({
+      type: Phaser.HEADLESS,
+    });
+  });
+
+  afterEach(() => {
+    game.phaseInterceptor.restoreOg();
+  });
+
+  beforeEach(() => {
+    game = new GameManager(phaserGame);
+    game.override
+      .battleType("single")
+      .startingLevel(100)
```

Comment:
```suggestion
      .startingLevel(100)
      .ability(Abilities.BALL_FETCH)
```
Test is flaky due to occasionally rolling No Guard Machamp.

Reply:
oh lol I was wondering about this, nice catch.